.github/workflows/workflow.yml

```
@@ -175,21 +175,11 @@ jobs:
     name: Update Brew Formula
     runs-on: macos-latest
```

It looks like we would no longer need to run this on `macos-latest` with this new action:
```suggestion
    runs-on: ubuntu-latest
```

---

Indeed 👍